src/devices/Board/GenericBoard.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Device.Gpio.Drivers;
+using System.Device.I2c;
+using System.Device.Pwm;
+using System.Device.Spi;
+using System.Text;
+
+namespace Iot.Device.Board
+{
+    /// <summary>
+    /// A generic board class. Uses generic implementations for GPIO, I2C etc
+    /// </summary>
+    public class GenericBoard : Board
```

Comment:
Is there a specific reason why this is public? I would rather have this (and the RaspberryPiBoard and CustomBoard) internal, and just have the Create method on the base class to create the right one for you. We could also even have an overload of Create that takes in an Enum so that we allow callers to specify which board they want to create. In the past on dotnet/iot we have found that using factory pattern is  better (like in SPI and I2C) for future refactorings since it was way easier to adapt and develop that when compared to GPIO (which doesn't have the factory pattern)

Reply:
This is the one you would use if you can't use or don't want to use the RaspberryPiBoard. It uses all the default implementations.